app/controllers/weeks_controller.rb

```diff
@@ -15,6 +15,27 @@ def show
         if (@rooms[0].state == "used" && @week.free_los(nil,current_user.id).count > 0)
           @rooms[0].update!(state: "do_not_use")
         end
+        # Ak sa nachadza viacero miestnosti na otvorenie necha len jednu alebo ziadnu ak uz nema nove learning objecty - boli odobrate
+        rooms_open = @rooms.select{|r| r.state == "do_not_use"}
+        if rooms_open.count > 1
+          j = rooms_open.count
+          if @week.free_los(nil,current_user.id).count == 0
+            j = j-1
+          else
+            j = j-2
+          end
+          for i in 0..j
```

Comment:
hm, tak to by som mohla spravit podobne aj to druhe ze z `rooms_open` zmazem jeden prvok a potom to iste resp najskor bud zmazem alebo nezmazem a potom vsetko nastavim na used